api/models/participant.py

```diff
@@ -2,10 +2,12 @@
 from django.conf import settings
 from django.core import mail
 from django.db import models
+from django.utils.translation import ugettext_lazy as _
```

Comment:
This usage is encouraged even in Python documentation: https://docs.python.org/3/library/gettext.htm

And I can't find any parameter, that You could feed to `makemessages` command to take else than `_()` as marked text when creating `.po` file.